pkg/image/decompress.go

```diff
@@ -26,12 +27,12 @@ func XzReader(r io.ReadCloser) (io.ReadCloser, error) {
 }
 
 func TarReader(r io.ReadCloser) (io.ReadCloser, error) {
-	glog.Infoln("TarReader: tar format")
+	glog.V(Vdebug).Infoln("TarReader: tar format")
 	tr := tar.NewReader(r)
 	hdr, err := tr.Next() // advance cursor to 1st (and only) file in tarball
 	if err != nil {
 		return r, fmt.Errorf("TarReader: reading tarfile header: %v\n", err)
 	}
-	glog.Infof("TarReader: extracting %q\n", hdr.Name)
+	glog.V(Vadmin).Infof("TarReader: extracting %q\n", hdr.Name)
```

Comment:
We could make the logs in this file more useful by adding something like this to each compression wrapper function and removing the simple `glog.V(Vdebug).Infoln("TarReader: tar format")` style logs.

Reply:
Did same for gz but xz does not have exported fields for name etc.